### DISCUSSION TITLE
Intermittent ORA-24459 (connection pooling errors) in production database

### DISCUSSION THREAD

**Discussion by jpstout** [godror/godror#290] [Q&A]
We are seeing errors within Star from the Godor Driver causing ORA-24459 (connection pooling errors). We have multiple logs and have contacted Oracle regarding this, and they see no issues on their side and have referred us over to Driver support.

Splunk log errors show this sort of issue:

ContractId: 626510538 IsRenewal: false Method: GetContractDetails RequestId: b2fc7ced-6ca7-48e8-af18-0037f94e8b6d TraceId: 226ba0ad9a0f439a72b6ca189c6775ae level: error msg: Internal : an unexpected error occurred : GetDetails : getRenewalsMonies : failed to query: ORA-24459: OCISessionGet() timed out waiting for pool to create new connections time: 05-25-2023 08:08:23.716 trace-id: 26b56fb1-9601-48dc-8e10-1698a3106664 } Show as raw text

{"ContractId":626510538,"IsRenewal":false,"Method":"GetContractDetails","RequestId":"b2fc7ced-6ca7-48e8-af18-0037f94e8b6d","TraceId":"226ba0ad9a0f439a72b6ca189c6775ae","level":"error","msg":"Internal : an unexpected error occurred : GetDetails : getRenewalsMonies : failed to query: ORA-24459: OCISessionGet() timed out waiting for pool to create new connections","time":"05-25-2023 08:08:23.716","trace-id":"26b56fb1-9601-48dc-8e10-1698a3106664"}

godror version: v0.36.0
Oracle 19c - Autonomous Shared on OCI

Please advise.

**Comment by tgulacsi** [comment]
How is the pool configured?
